app/controllers/datasets/links_controller.rb

```diff
@@ -60,8 +60,7 @@ def link_params
     params.require(:link).permit(
       :url,
       :name,
-      :start_day, :start_month, :start_year,
-      :end_day, :end_month, :end_year,
+      :day, :month, :year,
       :year,
```

Comment:
`year` and `quarter` are redundant, as they could be derived from day/month/year and the frequency of the dataset. It would make for a cleaner model to remove this redundancy. Year is trivial, you just need helper functions to convert quarters to date and vice-versa.